artemis-ra/src/main/java/org/apache/activemq/artemis/ra/inflow/ActiveMQActivationSpec.java

```diff
@@ -603,7 +596,11 @@ public void setMaxSession(final Integer value) {
          logger.trace("setMaxSession(" + value + ")");
       }
 
-      maxSession = value;
+      if ( value < 1 ) {
+         logger.warn("Invalid number of session (negative):" + value + ", defaulting to 1.");
```

Comment:
this requires a declared  logger in ActiveMQRALogger and logger code

Reply:
Quite sorry about that. I forgot about those. I'll fix it up right away.